tests/unit/channel_test.py

```diff
@@ -68,3 +72,82 @@ def test_read_bulk_response(ssh_client_session):
             break
 
     assert b'Hello World' in response  # noqa: WPS302
+
+
+def test_request_exec(ssh_channel):
+    """Test direct call to request_exec."""
+    ssh_channel.request_exec('exit 1')
+
+    rc = -1
+    while rc == -1:
+        ssh_channel.poll(timeout=5)
+        rc = ssh_channel.get_channel_exit_status()
+    assert rc == 1
+
+
+def test_send_eof(ssh_channel):
+    """Test send_eof correctly terminates input stream."""
+    ssh_channel.request_exec('cat')
+    ssh_channel.send_eof()
+
+    rc = -1
+    while rc == -1:
+        ssh_channel.poll(timeout=5)
+        rc = ssh_channel.get_channel_exit_status()
+    assert rc == 0
+
+
+def test_send_signal(ssh_channel):
+    """Test send_signal correctly forwards signal to the process."""
+    ssh_channel.request_exec('bash -c \'trap "exit 1" SIGUSR1; echo ready; sleep 5; exit 0\'')
+
+    # Wait until the process is ready to receive signal
+    output = ''
+    while not output.startswith('ready'):
+        ssh_channel.poll(timeout=1000)
+        output += ssh_channel.recv().decode('utf-8')
+
+    # Send SIGINT
+    ssh_channel.send_signal('USR1')
+
+    rc = -1
+    while rc == -1:
+        ssh_channel.poll(timeout=5)
+        rc = ssh_channel.get_channel_exit_status()
+
+    assert rc == 1
+
+
+def test_recv_eof(ssh_channel):
+    """
+    Test that reading EOF does not raise error.
+
+    SystemError: Negative size passed to PyBytes_FromStringAndSize
+    """
+    ssh_channel.request_exec('exit 0')
+    ssh_channel.poll(timeout=1000)
+    ssh_channel.recv()
+
+
+def test_is_eof(ssh_channel):
+    """Test that EOF-state is correctly obtained with is_eof."""
+    ssh_channel.request_exec('exit 0')
+    ssh_channel.poll(timeout=5000)  # noqa: WPS432
```

Comment:
Disabling this linting rule is not justified here. Assign a name to that arbitrary number of `5_000` that labels it with some context.